### PR TITLE
Zaier

### PR DESCRIPTION
**Pull request by ZaierAouani**
I am done with US206.

### CODE REVIEW

File: src/US205/zaier/US206.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package US205.zaier;
+
+public class US206 {
+
+    //Work 1
+    //more work 1
```
Comment (AnnaLoshak) [inline src/US205/zaier/US206.java:6]: please make some changes here

**Review by BadamkhandB — CHANGES_REQUESTED** [review]
Can you add one more work?

**Review by AnnaLoshak — CHANGES_REQUESTED** [review]

File: src/US205/zaier/US206.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
```
[the remark this answers is not in the document]
Comment (Aramat16ukr) [reply src/US205/zaier/US206.java:7]: needs correction

**Review by VeronikaKrasnokutska — APPROVED** [review]
Great job, keep going

**Review by Aramat16ukr — APPROVED** [review]
good job

**Review by Idemianiv — APPROVED** [review]
good job

File: src/US205/zaier/US206.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (natebell510) [reply src/US205/zaier/US206.java:7]: bueno

**Review by BadamkhandB — APPROVED** [review]

**Review by AnnaLoshak — APPROVED** [review]

**Review by natebell510 — COMMENTED** [review]
cat dog animal

**Pull request merged**